### ISSUE TITLE
getStatus missing in getContact/getChatById

### ISSUE THREAD

**Issue by t0g3pii** [open-wa/wa-automate-nodejs#2004]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Current Behavior

http://localhost:8080/getStatus ->
`{
  "success": true,
  "response": {
    "id": "000@c.us",
    "status": "🐢💞🤫 #Subby x #MyLittleOne 🌹~25-02-21~🌹"
  }
}`

http://localhost:8080/getChatById ->
`{
  "success": true,
  "response": {
    "id": "000@c.us",
    "pendingMsgs": false,
    "lastReceivedKey": {
      "fromMe": false,
      "remote": {
        "server": "c.us",
        "user": "000",
        "_serialized": "000@c.us"
      },
      "id": "0A0A",
      "_serialized": "false_000@c.us_0A0A"
    },
    "t": 1628810970,
    "unreadCount": 1,
    "archive": false,
    "isReadOnly": false,
    "muteExpiration": 0,
    "name": "t0g3pii",
    "notSpam": true,
    "pin": 0,
    "ephemeralDuration": 0,
    "ephemeralSettingTimestamp": 0,
    "msgs": null,
    "kind": "chat",
    "canSend": true,
    "isGroup": false,
    "formattedTitle": "t0g3pii",
    "contact": {
      "id": "000@c.us",
      "name": "t0g3pii",
      "shortName": "t0g3pii",
      "pushname": "𓆩ƬӨGΣPII𓆪",
      "type": "in",
      "isBusiness": false,
      "isEnterprise": false,
      "statusMute": false,
      "labels": [],
      "formattedName": "t0g3pii",
      "isMe": false,
      "isMyContact": true,
      "isPSA": false,
      "isUser": true,
      "isWAContact": true,
      "profilePicThumbObj": {
        "eurl": "img.jpg",
        "id": "000@c.us",
        "img": "img.jpg",
        "imgFull": "img.jpg",
        "raw": null,
        "tag": "1620472948"
      },
      "msgs": null
    },
    "groupMetadata": null,
    "presence": {
      "id": {
        "server": "c.us",
        "user": "000",
        "_serialized": "000@c.us"
      },
      "chatstates": []
    },
    "isOnline": true,
    "participantsCount": 1
  }
}`

### Expected Behavior

http://localhost:8080/getChatById ->
`{
  "success": true,
  "response": {
    "id": "000@c.us",
    "pendingMsgs": false,
    "lastReceivedKey": {
      "fromMe": false,
      "remote": {
        "server": "c.us",
        "user": "000",
        "_serialized": "000@c.us"
      },
      "id": "0A0A",
      "_serialized": "false_000@c.us_0A0A"
    },
    "t": 1628810970,
    "unreadCount": 1,
    "archive": false,
    "isReadOnly": false,
    "muteExpiration": 0,
    "name": "t0g3pii",
    "notSpam": true,
    "pin": 0,
    "ephemeralDuration": 0,
    "ephemeralSettingTimestamp": 0,
    "msgs": null,
    "kind": "chat",
    "canSend": true,
    "isGroup": false,
    "formattedTitle": "t0g3pii",
    "contact": {
      "id": "000@c.us",
      "name": "t0g3pii",
      "shortName": "t0g3pii",
      "pushname": "𓆩ƬӨGΣPII𓆪",
      "status": "🐢💞🤫 #Subby x #MyLittleOne 🌹~25-02-21~🌹",
      "type": "in",
      "isBusiness": false,
      "isEnterprise": false,
      "statusMute": false,
      "labels": [],
      "formattedName": "t0g3pii",
      "isMe": false,
      "isMyContact": true,
      "isPSA": false,
      "isUser": true,
      "isWAContact": true,
      "profilePicThumbObj": {
        "eurl": "img.jpg",
        "id": "000@c.us",
        "img": "img.jpg",
        "imgFull": "img.jpg",
        "raw": null,
        "tag": "1620472948"
      },
      "msgs": null
    },
    "groupMetadata": null,
    "presence": {
      "id": {
        "server": "c.us",
        "user": "000",
        "_serialized": "000@c.us"
      },
      "chatstates": []
    },
    "isOnline": true,
    "participantsCount": 1
  }
}`

### Steps To Reproduce

-

### Mode

EASY API/CLI

### create() code

```markdown
npx @open-wa/wa-automate -p 8080 -k 'PASSWORD' -s "SESSION_DATA=" --keep-alive
```

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.2130.9",
  "WA_AUTOMATE_VERSION": "4.14.0",
  "BROWSER_VERSION": "HeadlessChrome/93.0.4577.0",
  "START_TS": 1628810861543
}
```

### Environment

```markdown
- OS: Windows 10
- Node: 12.22.1
- npm: 7.20.5
```

**Comment by smashah** [comment]
@t0g3pii was the status ever there in the first place?

**Comment by t0g3pii** [comment]
> @t0g3pii was the status ever there in the first place?

no, but would love to see it there^^ currently i need to do 2 API Calls instead of one just for the Status

**Comment by smashah** [comment]
@t0g3pii this patch has been reverted due to adverse side-effects :(
